### PR TITLE
iwgtk: update to 0.8, adopt

### PR DESCRIPTION
**Pull request by subnut**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by subnut** [comment]
@abenson Is it okay if I adopt this package?

**Comment by classabbyamp** [comment]
the maintainer is actually @caughtquick

**Comment by caughtquick** [comment]
@classabbyamp thanks for the ping @subnut feel free to adopt this package, I don't use void anymore

File: srcpkgs/iwgtk/template [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-homepage="https://github.com/J-Lentz/iwgtk/"
+homepage="https://github.com/J-Lentz/iwgtk"
+changelog="https://raw.githubusercontent.com/J-Lentz/iwgtk/master/CHANGELOG"
 distfiles="https://github.com/J-Lentz/iwgtk/archive/refs/tags/v${version}.tar.gz"
-checksum=71bb85546a55bf710052d4947f6be5f5a01033d04d2d14a85a7e734dd570657d
+checksum=38e81e67004bbd246b6551ffc812b6ab019f84a7329f134bb8031eb61d9d732c
+
+post_install() {
+	# Remove systemd files
+	rm -rf "${DESTDIR}/usr/share/lib/systemd"
```
Comment (classabbyamp) [inline srcpkgs/iwgtk/template:19]: https://github.com/void-linux/void-packages/blob/master/Manual.md#writing-runit-services

> If the package installs a systemd service file or other unit, leave it in place as a reference point so long as including it has no negative side effects.
> 
> Examples of when not to install systemd units:
> 
> 1. When doing so changes runtime behavior of the packaged software.
> 2. When it is done via a compile time flag that also changes build dependencies.

**Pull request merged**